tests/pytests/test.py

```diff
@@ -4305,3 +4305,53 @@ def test_notIterTimeout(env):
         'FT.AGGREGATE', 'idx', '-@tag1:{fantasy}', 'LOAD', '2', '@title', '@n',
         'APPLY', '@n^2 / 2', 'AS', 'new_n', 'GROUPBY', '1', '@title', 'TIMEOUT', '1'
     ).error().contains('Timeout limit was reached')
+
+@skip(cluster=False, min_shards=2)
+def test_incompatibleIndex(env):
+    """Tests that we get an error if we try to query an index with a different
+    schema than the one used in the query"""
+
+    # Connect to two shards
+    first_conn = env.getConnection(0)
+    second_conn = env.getConnection()
```

Comment:
isn't that the same shard? can we do
```suggestion
    first_conn = env.getConnection(0)
    second_conn = env.getConnection(1)
```
to be more explicit?

Reply:
This is 100% what is was supposed to be 🤔